.github/workflows/example-ktx.yml

```diff
@@ -0,0 +1,35 @@
+name: Android CI
```

Comment:
It should have a different name, so we can distinguish it in the GitHub Actions UI